src/state/selectors/layers.js

```diff
@@ -1,7 +1,8 @@
 import { createSelector } from 'reselect';
-import MiradorCanvas from '../../lib/MiradorCanvas';
+import { getMiradorCanvasWrapper } from './wrappers';
 import { getCanvas, getVisibleCanvasIds } from './canvases';
-import { miradorSlice } from './utils';
+import { miradorSlice, EMPTY_ARRAY } from './utils';
+import { getConfig } from './config';
```

Comment:
getConfig import is not used in this file

Reply:
that's weird - why didn't the linter catch this?